README.md

```diff
@@ -11,7 +38,11 @@ To run locally you'll need a `.env` file with these properties:
 export SLACK_SIGNING_SECRET=
 export SLACK_CLIENT_ID=
 export SLACK_CLIENT_SECRET=
+<<<<<<< HEAD
```

Comment:
Looks like some merge conflict leftovers

Reply:
whoops

Reply:
I forgot to save my editor 🤦